src/Cooling/test_cooling.cpp

```diff
@@ -39,7 +39,16 @@ template <> struct HydroSystem_Traits<CoolingTest> {
   static constexpr double gamma = 5. / 3.; // default value
   // if true, reconstruct e_int instead of pressure
   static constexpr bool reconstruct_eint = true;
-  static constexpr int nscalars = 0;       // number of passive scalars
+};
+
+template <> struct Physics_Traits<CoolingTest> {
+  static constexpr bool is_hydro_enabled = true;
+  static constexpr bool is_radiation_enabled = false;
+  static constexpr bool is_mhd_enabled = false;
```

Comment:
Can you make it clear in the comments that setting `is_mhd_enabled` to true doesn't actually do anything yet? Alternatively, we could just add `is_mhd_enabled` in a future MHD-specific PR.

Reply:
You could add a `static_assert` in the RadhydroSimulation constructor to ensure that `is_chemistry_enabled` and `is_mhd_enabled` are set to false, just to make sure users don't try to active them yet.

Reply:
Yes, I think it would make sense to remove all references to is_mhd_enabled, and then I will add it in the mhd-branch

Reply:
Also, yep, I'll add the assert.

Reply:
Done.